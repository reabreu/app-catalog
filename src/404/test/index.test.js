import React from "react";
import { render } from "@testing-library/react";
import NotFound from "../index";

describe("<NotFound /> spec", () => {
  it("Renders NotFound", () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
