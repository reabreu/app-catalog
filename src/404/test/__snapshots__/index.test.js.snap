// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> spec Renders NotFound 1`] = `
<DocumentFragment>
  <p>
    Sorry but the page you're looking for does not exist
  </p>
</DocumentFragment>
`;
